packages/react-app-polyfill/ie9.js

```diff
@@ -11,4 +11,7 @@ require('./ie11');
 // React 16+ relies on Map, Set, and requestAnimationFrame
 require('core-js/features/map');
 require('core-js/features/set');
-require('raf').polyfill(window);
+
+if (typeof window !== 'undefined') {
+  require('raf').polyfill(window);
```

Comment:
It seems like we can instead use `require('raf').polyfill()`? It will infer the global object it looks like: https://github.com/chrisdickinson/raf#rafpolyfillobject.